packages/@react-spectrum/datepicker/src/styles.css

```diff
@@ -111,22 +111,13 @@
   }
 }
 
-.react-spectrum-DatePicker-placeholder {
-  display: block;
-  width: 100%;
-  text-align: center;
-  font-style: italic;
-  visibility: hidden;
-  height: 0;
-  pointer-events: none;
-}
-
 .react-spectrum-DatePicker-cell.is-placeholder {
   color: var(--spectrum-gray-600);
 
   .react-spectrum-DatePicker-placeholder {
-    visibility: visible;
-    height: auto;
+    text-align: center;
+    font-style: italic;
```

Comment:
This element isn't working at all right now so I'm not seeing the italics. But I guess we should get rid of them anyway?

Reply:
Ditto the above for not seeing italics.
We are purging italics, but do we need that visual indicator for placeholder?